Orange/canvas/application/workflows/__init__.py

```diff
@@ -32,55 +30,57 @@ def default_entry_point():
     return ep
 
 
-def tutorial_entry_points():
-    """Return an iterator over all tutorials.
+def workflow_entry_points():
+    """Return an iterator over all example workflows.
     """
     default = default_entry_point()
     return chain([default],
-                 pkg_resources.iter_entry_points("orange.widgets.tutorials"))
+                 pkg_resources.iter_entry_points("orange.widgets.workflows"))
```

Comment:
This breaks all add-ons that register their own tutorials/workflows using this entry point.